packages/ui/packages/Carousel/test/utils.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
if you fixed the lint , why should disable it?

Reply:
I should install eslint-plugin-jest  for test lint, eslint not support without this plugin

Reply:
I add this plugin and fix